photos.html

```diff
@@ -1 +1,229 @@
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
```

Comment:
The margin on the right is larger than the margin on the left. This is probably because the images don't fit the container 100%, and are aligned to the top-left corner of its container, resulting in a larger right margin.

Suggested fix: min-width and min-height to 100%